source-code/website/src/pages/documentation/EditButton.tsx

```diff
@@ -0,0 +1,17 @@
+interface EditButtonProps {
+	href: string | undefined
+}
+
+export function EditButton(props: EditButtonProps) {
+	return (
+		<div class="md:pl-9 w-full pt-8 sm:pt-12 max-sm:pb-8">
+			<a
+				href={props.href}
+				class="text-info/80 hover:text-info/100 text-sm font-semibold"
+				target="_blank"
+			>
+				Edit on GitHub
```

Comment:
What about adding an icon as a prefix? I think that makes in more clearly clickable. And directly implies what it is. 
https://icon-sets.iconify.design/material-symbols/edit-outline-rounded/ maybe this one. U should be able to import the material symbols like the other icons.